Makefile

```diff
@@ -450,7 +450,7 @@ create-management-cluster: $(KUSTOMIZE) $(ENVSUBST)
 
 	# Deploy CAPI
 	# Using cluster-api nightly build until 0.4.x is released.
-	curl --retry $(CURL_RETRIES) -sSL https://storage.googleapis.com/artifacts.k8s-staging-cluster-api.appspot.com/components/nightly_master_20210210/cluster-api-components.yaml | $(ENVSUBST) | kubectl apply -f -
+	curl --retry $(CURL_RETRIES) -sSL https://storage.googleapis.com/artifacts.k8s-staging-cluster-api.appspot.com/components/nightly_master_20210301/cluster-api-components.yaml | $(ENVSUBST) | kubectl apply -f -
```

Comment:
Any way we can do this programmatically rather than hard coding?

Reply:
I was thinking we should have a tag called `nightly_latest` or something like that, but then you never know if some type changed and could break things

how would be do this programmatically, we don't know the last successful tag, or are you thinking we can check latest in gcr?

Reply:
I wasn't sure on the how more questioning if there is a how. 

We know the format. Could we try today, then say 10 of the previous days to find the first that is successful? Fail if we don't find one.

Reply:
I think there is value in pinning to a specific version so we can deliberately update the version when we're ready to ingest new changes, especially since changes tend to be breaking these days.

Reply:
Moar GitHub points!

Reply:
> Moar GitHub points!

does that mean you concur @devigned? 😄

Reply:
Indeed. Seems reasonable enough to me. We'll just have to keep ourselves honest and keep moving forward as we can.